package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "lemmy-ui",
-  "version": "0.17.1",
+  "version": "0.17.3",
   "description": "An isomorphic UI for lemmy",
-  "repository": "https://github.com/LemmyNet/lemmy-ui",
```

Comment:
Why did this get removed?

Reply:
It could be another error that snuck in during the merge.